SofaKernel/modules/SofaCore/src/sofa/core/ObjectFactory.h

```diff
@@ -91,12 +91,16 @@ class SOFA_CORE_API ObjectFactory
 
         std::string className;
         std::set<std::string> aliases;
+        std::set<std::string> deprecatedAliases;
         std::string description;
         std::string authors;
         std::string license;
         std::string defaultTemplate;
+        std::string compilation_target;
```

Comment:
```suggestion
        std::string compilationTarget;
```
To follow guidelines and keep consistency with other class member names.